### ISSUE TITLE
Change existing label for a text an re-train the model

### ISSUE THREAD

**Issue by eshbeata** [fatiherikli/klassify#7]
Hi

In some cases the data change and we would need to change the label of a text , this should be a cool feature to have .

Support edit and delete for a text also would be good to have.
Thanks

**Comment by randomm** [comment]
Label is part of the redis key, so I think it should be a relatively easy operation to a) rename the key, and b) rename the label in the labels collection (change the key value inside the collection). http://redis.io/commands/rename

**Comment by eshbeata** [comment]
Yes but my issue is not the label key , what im saying is as below :

Consider the following :
label=art  text=Lorem Ipsum is simply dummy text of the printing and typesetting industry.
Now the classification well train the model under the art label .

So the label was assign incorrectly for that text and the correct label should be sports
label=sports   text=Lorem Ipsum is simply dummy text of the printing and typesetting industry.

Of course we can post the train request again but still updating the data should the correct thing to do in order to train the model again with the correct label .

**Comment by randomm** [comment]
You mean change the label for a particular piece of taught text, not the complete text hash (that could include N rounds of teaching)? I'm not sure what would be the best course of action for that, however, adding functionality to rename labels (as I described above) would probably be quite easily doable by adding another endpoint for that and doing some redis operations.
